webpage/templates/source_detail.txt

```diff
@@ -22,7 +32,7 @@ Lightcurves
 -----------
 
 {% for resource in resources.resources if resource.type == 'lightcurve' %}
-* `{{ resource.reference_id }} <../../output/{{ resource.url }}>`__
+* {{ resource.reference_id }} (`Show <https://github.com/gammapy/gamma-cat/tree/master/output/{{ resource.url }}>`__, `Download <../../output/{{ resource.url }}>`__, `Gamma-Cat Input <https://github.com/gammapy/gamma-cat/tree/master/input/data/{{ resource.year }}/{{ resource.parsedref }}/tev-{{ resource.source_str }}-lc.ecsv>`__)
```

Comment:
Please break these long lines in the template, putting each url on a separate line, to make it more readable / maintainable.

Also, it would be better to pass a "github_base_url" to the template and render it, instead of to repeat the same base URL many times in the template.